uaclient/messages.py

```diff
@@ -731,6 +731,13 @@ class TxtColor:
     "Cannot install Real-Time Kernel on a container.",
 )
 
+GCP_SERVICE_ACCT_NOT_ENABLED_ERROR = NamedMessage(
+    "gcp-pro-service-account-not-enabled",
+    "Failed to attach machine\n{error_msg}",
+)
+
+GCP_SERVICE_ACCT_URL_MESSAGE = """\
+For more information, see https://cloud.google.com/iam/docs/service-accounts"""
```

Comment:
Do these have to be two separate strings?
```suggestion
GCP_SERVICE_ACCT_NOT_ENABLED_ERROR = NamedMessage(
    "gcp-pro-service-account-not-enabled",
    "Failed to attach machine\n"
    "{error_msg}\n"
    "For more information, see https://cloud.google.com/iam/docs/service-accounts",
)

Reply:
I thought it would be a line to use later, but I guess we will refactor later when we need it separated.